components/studio/bin/hab-studio.ps1

```diff
@@ -301,12 +301,22 @@ function Enter-Studio {
         Start-Process "$env:STUDIO_SCRIPT_ROOT\powershell\pwsh.exe" -ArgumentList "-Command `"& {Get-Content $env:HAB_STUDIO_ENTER_ROOT\hab\sup\default\out.log -Tail 100 -Wait}`""
       }
       else {
-        Get-Content $env:HAB_STUDIO_ENTER_ROOT\hab\sup\default\out.log -Tail 100 -Wait
```

Comment:
Right, this would have blocked here?

Reply:
yeah which is actually the right thing to do. Its just like a `tail -f` in bash requiring a `ctrl+c` to quit.